server/api/batch_inventory.py

```diff
@@ -116,19 +135,23 @@ def parse_vote(vote: str):
         batch_to_counting_group[batch_key] = counting_group
 
         choice_votes = {
-            choice.name: parse_vote(
+            choice_name: parse_vote(
                 column_value(
-                    row, choice.name, cvr_number, choice_indices, required=False
+                    row, choice_name, cvr_number, choice_indices, required=False
                 )
             )
-            for choice in contest.choices
+            for choice_name in choice_indices.keys()
```

Comment:
Curious why we changed from `contest.choices` to `choice_indices.keys()` here (I may be missing something obvious!)